assets/src/stories-editor/helpers/index.js

```diff
@@ -243,12 +243,10 @@ export const addAMPAttributes = ( settings, name ) => {
 
 	if ( isMovableBlock ) {
 		addedAttributes.positionTop = {
-			type: 'number',
```

Comment:
Is number only for integers? Or why was this removed?

Reply:
It looks like it was saving it as a string. Having a type shouldn't be mandatory anyway, at some point adding a type even caused a warning, probably it has changed by now but AFAIK it's still not necessarily needed. Thoughts?

Reply:
Hmm, according to the docs "the framework does not automatically perform type casting of meta", so if you expect numbers you'd need to do that yourselves. Would have to check the actual code to see what it does though.

Just wanted to point it out here though as it seemed to be a bit of an exception.

Reply:
Hmm, the logic is using `.toFixed()`, probably that's why it's saving as a `string`.
For saving as a number we'd have to convert it again with `parseFloat` then perhaps.

Reply:
Yeah that would need a type cast again.

Reply:
OK, let's cast it to a number then.

Reply:
Added within 9bc8981.
Also added back the type for readability.